commons/utils/s3_utils.py

```diff
@@ -389,3 +390,10 @@ def assert_s3_err_msg(rgw_error, cortx_error, cmn_cfg, error):
         assert_utils.assert_in(rgw_error, error.message, error.message)
     else:
         assert_utils.assert_in(cortx_error, error.message, error.message)
+
+
+def create_random_file(file_path, min_size, max_size):
```

Comment:
can't we use create_file() function

Reply:
Moved existing code from s3_test_lib here, so retained the functionality as is. Based on our offline discussion I will compare the performance of both this approach and create_file and update accordingly in test execution follow up task,